0.6-dev/index.bs

```diff
@@ -0,0 +1,1787 @@
+- Parameter values MUST be compatible with input and output space dimensionality (see details).
+
+<table>
+  <tr><th>`identity`
+    <td> 
+    <td>The identity transformation is the default transformation and is typically not explicitly defined.
+  <tr><th>`mapAxis`
+    <td>`"mapAxis":Dict[String:String]`
+    <td> A `maxAxis` transformation specifies an axis permutation as a map between axis names.
+  <tr><th>`translation`
+    <td> one of: <br>`"translation":List[number]`, <br>`"path":str`
+    <td>translation vector, stored either as a list of numbers (`"translation"`) or as binary data at a location
+    in this container (`path`).
+  <tr><th>`scale`
+    <td> one of: <br>`"scale":List[number]`, <br>`"path":str`
+    <td>scale vector, stored either as a list of numbers (`scale`) or as binary data at a location in this
+    container (`path`).
+  <tr><th>`affine`
+    <td> one of: <br>`"affine":List[List[number]]`, <br>`"path":str`
+    <td>affine transformation matrix stored as a flat array stored either with json uing the affine field
+    or as binary data at a location in this container (path). If both are present, the binary values at path should be used.
+  <tr><th>`rotation`
+    <td> one of: <br>`"rotation":List[number]`, <br>`"path":str`
+    <td>rotation transformation matrix stored as an array stored either
+        with json or as binary data at a location in this container (path).
+        If both are present, the binary parameters at path are used.
+  <tr><th>`sequence`
+    <td> `"transformations":List[Transformation]`
+    <td>A sequence of transformations, Applying the sequence applies the composition of all transforms in the list, in order.
+  <tr><th>`displacements`
+    <td>`"path":str`<br>`"interpolation":str`
+    <td>Displacement field transformation located at (path).
+  <tr><th>`coordinates`
+    <td>`"path":str`<br>`"interpolation":str`
+    <td>Coordinate field transformation located at (path).
+  <tr><th>`inverseOf`
+    <td>`"transform":Transform`
+    <td>The inverse of a transformation. Useful if a transform is not closed-form invertible. See Forward and inverse for details and examples.
+  <tr><th>`bijection`
+    <td>`"forward":Transform`<br>`"inverse":Transform`
+    <td>Explicitly define an invertible transformation by providing a forward transformation and its inverse.
+  <tr><th>`byDimension`
+    <td>`"transformations":List[Transformation]`
+    <td>Define a high dimensional transformation using lower dimensional transformations on subsets of
+    dimensions.
+ <thead>
+   <tr><th>type<th>fields<th>description
+</table>
+
+
+Conforming readers:
+- MUST parse `identity`, `scale`, `translation` transformations;
+- SHOULD parse `mapAxis`, `affine` transformations;
+- SHOULD be able to apply transformations to points;
+- SHOULD be able to apply transformations to images;
+
+Coordinate transformations from array to physical coordinates MUST be stored in multiscales ([[#multiscale-md]]). 
```

Comment:
this sentence implies that coordinate transformations from array to non-physical coordiantes (from array to array coordinates, I guess?) could be stored somewhere other than `multiscales`. Is that the intention?